examples/src/main/java/io/dapr/examples/configuration/grpc/ConfigurationClient.java

```diff
@@ -106,36 +110,88 @@ public static void getConfigurations(DaprPreviewClient client) {
    *
    * @param client DaprPreviewClient object
    */
-  public static void subscribeConfigurationRequestWithSubscribe(DaprPreviewClient client) {
-    System.out.println("*****Subscribing to keys using subscribe method: " + keys.toString() + " *****");
+  public static void subscribeConfigurationRequest(DaprPreviewClient client) {
+    System.out.println("Subscribing to key: myconfig1");
     AtomicReference<Disposable> disposableAtomicReference = new AtomicReference<>();
-    SubscribeConfigurationRequest req = new SubscribeConfigurationRequest(CONFIG_STORE_NAME, keys);
+    SubscribeConfigurationRequest req = new SubscribeConfigurationRequest(
+        CONFIG_STORE_NAME, Collections.singletonList("myconfig1"));
     Runnable subscribeTask = () -> {
-      Flux<Map<String, ConfigurationItem>> outFlux = client.subscribeToConfiguration(req);
+      Flux<SubscribeConfigurationResponse> outFlux = client.subscribeConfiguration(req);

```

Comment:
I think this is too complicated to use this feature in the SDK. We should follow up with a solution that wraps the update of the key into a wrapper class: DaprSubscribedConfig or something. Then, the app just needs to get the value and it is automatically updated behind the scenes. The app can also register a callback function in that instance to receive a callback when it is updated.

Reply:
@artursouza In reality the useful code for subscribing is only this much. 

```
Flux<SubscribeConfigurationResponse> outFlux = client.subscribeConfiguration(req);
outFlux.subscribe(
        cis -> {
          cis.getItems().forEach((k,v) -> print(v,k));
        }
 ));
``` 
rest other things are are for demonstrating how this is going to work when there is some config update going on behind the scene.
I have simplified the both http and gRPC examples now.

Reply:
Yes. I think we can make it even more convenient. I will propose something for 1.10